upgrade/sql/8.0.0.sql

```diff
@@ -23,7 +23,8 @@ INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `po
   (NULL, 'actionPresentPaymentOptions', 'Payment options Presenter', 'This hook is called before payment options are presented', '1'),
   (NULL, 'actionCustomerLogoutBefore', 'Before customer logout', 'This hook allows you to execute code before customer logout', '1'),
   (NULL, 'actionCustomerLogoutAfter', 'After customer logout', 'This hook allows you to execute code after customer logout', '1'),
-  (NULL, 'displayCheckoutSummaryTop', 'Cart summary top', 'This hook allows you to display new elements in top of cart summary', '1')
+  (NULL, 'displayCheckoutSummaryTop', 'Cart summary top', 'This hook allows you to display new elements in top of cart summary', '1'),
+  (NULL, 'actionExportOrder', 'Allows you to modify the csv generated by the order export in the administration', 'This hook is executed before the csv output of the commands', '1')
```

Comment:
Hi @olecorre  

seems like this update is related to this PR which was not merged in the 8.0 https://github.com/PrestaShop/PrestaShop/pull/27885

Since the branch has been frozen it won't be integrated in the 8.0 but in the 8.1, so you're gonna need to change this PR to target the `8.1.0.sql` file (which doesn't exist yet).